hc/front/views.py

```diff
@@ -575,3 +575,55 @@ def failed_checks(request):
     }
 
     return render(request, "front/failed_checks.html", ctx)
+
+
+@login_required
+def blog(request):
+    b = Blog.objects.filter(user=request.team.user).order_by("created")
+    blogs = list(b)
```

Comment:
`querysets` are iterable. We should be able to iterate over them without explicitly converting to a list